src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/RestoreAzureRMSqlDatabase.cs

```diff
@@ -122,15 +122,18 @@ public class RestoreAzureRmSqlDatabase
         /// </summary>
         [Parameter(Mandatory = false,
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The database edition to use for the restored database.")]
+            HelpMessage = "The database edition to use for the restored database." +
+            "New supported database edition: GeneralPurpose and BusinessCritical")]
```

Comment:
This addition is not needed. A list of editions is retrieved by looking at the DatabaseEdition object.

Reply:
@akromm yes, will fix.